tests/Functional/Admin/PantherTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\PageBundle\Tests\Functional\Admin;
+
+use DAMA\DoctrineTestBundle\Doctrine\DBAL\StaticDriver;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\Tools\SchemaTool;
+use Sonata\PageBundle\Tests\App\Entity\SonataPageSite;
+use Symfony\Component\Panther\PantherTestCase;
+
+final class PantherTest extends PantherTestCase
+{
+    public static function setUpBeforeClass(): void
+    {
+        StaticDriver::setKeepStaticConnections(false);
+
+        self::prepareData();
+    }
+
+    public static function tearDownAfterClass(): void
+    {
+        self::rollbackDatabase();
+
+        StaticDriver::setKeepStaticConnections(true);
+    }
```

Comment:
This allows us to bypass DAMA Doctrine test bundle and load data specifically for this test class. WDYT @VincentLanglet ? It looks good enough for me.

Reply:
Yeah seems nice